components/org.wso2.carbon.identity.scim2.common/src/test/java/org/wso2/carbon/identity/scim2/common/impl/SCIMUserManagerTest.java

```diff
@@ -803,4 +812,188 @@ public void testUpdateUserWithUsernameChange() throws Exception {
 
         assertTrue("UserName claim update is not properly handled.", hasExpectedBehaviour);
     }
+
+    @Test
+    public void testGetUserSchema() throws CharonException, ClaimMetadataException {
+        String tenantDomain = "carbon.super";
+        String externalClaimURI = "urn:ietf:params:scim:schemas:core:2.0:User";
+
+        Map<String, String> properties = new HashMap<String, String>() {{
+            put("ReadOnly", "true");
```

Comment:
must be have constants for these properties already. If not lets create a constant.